pkg/config/config.go

```diff
@@ -12,6 +13,8 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const defaultMaxNumTracks = 4000
```

Comment:
The comment in `config-sample.yaml` says the default is 200.

Reply:
Oh wait, I see.  200 per CPU.

Still, if there is to be a hard max it should probably be mentioned in the comment.